server/src/main/java/org/elasticsearch/action/admin/cluster/settings/ClusterUpdateSettingsResponse.java

```diff
@@ -68,4 +73,31 @@ public void writeTo(StreamOutput out) throws IOException {
         Settings.writeSettingsToStream(persistentSettings, out);
         writeAcknowledged(out);
     }
+
+    @Override
+    public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
+        builder.startObject();
+        addAcknowledgedField(builder);
+        builder.startObject("persistent");
+        persistentSettings.toXContent(builder, params);
+        builder.endObject();
+        builder.startObject("transient");
+        transientSettings.toXContent(builder, params);
+        builder.endObject();
+        builder.endObject();
+        return builder;
+    }
+
+    private static final ObjectParser<ClusterUpdateSettingsResponse, Void> PARSER = new ObjectParser<>("clustre_update_settings_response",
+            true, ClusterUpdateSettingsResponse::new);
+
+    static {
+        PARSER.declareBoolean((r, a) -> r.acknowledged = a, new ParseField("acknowledged"));
```

Comment:
call `declareAcknowledgedField` ? then you would not need to make the field protected in the super class?